example/spec/parallel/distributionAPI/distributionSpec.js

```diff
@@ -24,7 +24,7 @@ const s3Data = [
   '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met',
 ];
 
-describe('Distribution API', () => {
+describe('TEA Distribution API', () => {
```

Comment:
I think we should probably rename this file to something like distributionTEASpec

Reply:
renamed